x-pack/plugins/infra/public/containers/logs/log_highlights/log_summary_highlights.ts

```diff
@@ -45,7 +49,10 @@ export const useLogSummaryHighlights = (
         setLogSummaryHighlights(response.data);
       },
     },
-    [sourceId, start, end, bucketSize, filterQuery, highlightTerms]
+    // Use `*Timestamp` values in the fetch.
+    // Use `*Date` to decide if it should refetch or not. `endTimestamp` updates
+    // frequently when `endDate` is `"now"` and triggers a lot of re-renders.
+    [sourceId, startDate, endDate, filterQuery, highlightTerms]
```

Comment:
Wouldn't the removal of the `bucketSize` mean it could use an outdated value here?

Reply:
In principle no since its value is linked to the range. If the range changes, the bucketSize changes as well.